src/index.js

```diff
@@ -371,7 +371,7 @@ export default class Drawer extends Component {
           return (
             <div
               id={id}
-              style={{ backgroundColor: `rgba(55, 56, 56, ${open ? 0.6 : 0})`, overflowX: 'hidden' }}
+              style={{ backgroundColor: `rgba(55, 56, 56, ${open ? 0.6 : 0})`, overflowX: isDirectionRight(direction) ? 'hidden' : 'visible'}}
```

Comment:
might be easier to just construct that style object in the render function like this

```
let styles = {
	backgroundColor: `rgba(55, 56, 56, ${open ? 0.6 : 0})`,
}

if (isDirectionRight(direction)) {
styles = {
 ...styles,
`overflowX: 'hidden'
}

```

Reply:
that way there is no fingerprint on the style object for people who aren't using this option.

does that make sense?

Reply:
You're right, maybe it's prettier :) it should be ok now. Thank you @hanford